openslides_backend/action/actions/meeting/create.py

```diff
@@ -47,6 +51,14 @@ class MeetingCreate(CreateActionWithDependencies, MeetingPermissionMixin):
     skip_archived_meeting_check = True
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        # update Translator
+        if instance.get("language") is None:
+            organization = self.datastore.get(
+                ONE_ORGANIZATION_FQID, ["default_language"], lock_result=False
+            )
+            instance["language"] = organization.get("default_language")
```

Comment:
you can remove this and make it required in the payload

Reply:
I have made "language" a required payload of meeting create and updated the tests.